cmd/agent/dist/conf.d/cisco_sdwan.d/conf.yaml.example

```diff
@@ -0,0 +1,76 @@
+## This file is overwritten upon Agent upgrade.
+## To make modifications to the check configuration, please copy this file
+## to `conf.yaml` and make your changes on that file.
```

Comment:
Should we mention that the integration is in beta?

Reply:
Since it will be use by customer I think this worth to say it here or in a release note with reno ?

Reply:
+1, let's make it clear in that file